Sources/Stats.php

```diff
@@ -451,6 +450,10 @@ function DisplayStats()
 	while ($row_members = $smcFunc['db_fetch_assoc']($members_result))
 	{
 		$i = array_search($row_members['id_member'], array_keys($members));
+		// skip all not top 10
+		if ($i >= 10)
+			continue;
```

Comment:
You might as well use `break` instead of `continue` here.

Reply:
Since the data is not sorted,
Reason why ksort is called later on,
Is continue the right call.